rails/spec/policies/admin/cohort_policy_spec.rb

```diff
@@ -50,4 +50,30 @@
       end
     end
   end
-end
\ No newline at end of file
+
+  describe 'create' do
+    let(:cohort_stubs) { {project: 'project' } }
+    let(:proj_user) { @cohort1 = FactoryBot.create(:user) }
```

Comment:
This is confusing it is creating a user object and assigning it to a variable called `@cohort1`. Above `@cohort1` is a admin_cohort not a user.

Reply:
Ooops, copy-pasta.  :p